scripts/istanbul.yml

```diff
@@ -1,5 +1,6 @@
 instrumentation:
-  include-all-sources: true
+# TODO: disabled all sources as istanbul crashes on `import` token in client code
+#  include-all-sources: true
```

Comment:
Seeing this made me create #850 for a fix, btw. Not perfect, but at least `npm run coverage` runs.